CONTRIBUTING.md

```diff
@@ -296,6 +296,113 @@ OS and is built by Android NDK. This often introduces compatibility issues as
 Android (specifically Termux) is not a standard platform. Do not expect there
 are exist package recipes available out-of-box.
 
+## Commit guidelines
+
+A commit message should look something like:
+```
+<commitType>(<repo>/<package>): (The change made/Short description of the change)
```

Comment:
Is this repo or branch? I see that "main" is used occasionally...

Reply:
repo/name, main, root or x11 (which is what they are called in [repo.json](https://github.com/termux/termux-packages/blob/master/repo.json)), but yes, could be clarified

Reply:
Thanks, mentioned the same in the guidelines as well in order to avoid confusion.

Hence resolving